bucket/grepwin.json

```diff
@@ -0,0 +1,82 @@
+{
+    "version": "1.9.1",
+    "description": "Regular expression search and replace tool for Windows.",
+    "homepage": "https://tools.stefankueng.com/grepWin.html",
+    "license": "GPL-3.0-only",
+    "notes": "Run 'grepwin-add-right-click' to add grepWin to right click menu options. \n(This does not require admin rights)",
+    "architecture": {
+        "32bit": {
+            "url": "https://github.com/stefankueng/grepWin/releases/download/1.9.1/grepWin-1.9.1_portable.exe#/grepWin_portable.exe",
+            "hash": "015a9427d29be42dfa0defc73aa613a44d6eb3b0f5b76152f988d941fb8d9868"
+        },
+        "64bit": {
+            "url": "https://github.com/stefankueng/grepWin/releases/download/1.9.1/grepWin-x64-1.9.1_portable.exe#/grepWin_portable.exe",
+            "hash": "95335b4ad1e42149a8e5e3ce4ce7114ba8cc50d5d671209a917c4aa6b8eca653"
+        }
+    },
+    "installer": {
```

Comment:
I would rather see more maintainable approach similar to vscode / tortoisesvn

Reply:
The solution from `VScode` looks much cleaner. I'll modify the manifest later.

Reply:
all done.

Reply:
Resolve conversations!